mrq/job.py

```diff
@@ -153,6 +155,7 @@ def set_data(self, data):
             task_def = self.get_task_config()
 
             self.timeout = task_def.get("timeout", cfg["default_job_timeout"])
+            self.default_ttl = task_def.get("default_ttl")
```

Comment:
Should we also have a cfg["default_job_ttl"] ?

Reply:
Yes, It can be useful to easily clean old jobs and better for consistency.